.github/labeler.yml

```diff
@@ -200,4 +200,4 @@
     - "!subsys/bluetooth/mesh/**/*"
 
 "no-changelog":
```

Comment:
@carlescufi , can I update the label name to "update-changelog" or "add-changelog"? I see lots of PRs being merged with no changes to changelog, and the name of the label could bear some blame here...

Reply:
Yes, you can do that no problem